src/cascadia/TerminalApp/userDefaults.json

```diff
@@ -6,6 +6,8 @@
 
     "defaultProfile": "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
 
+    "confirmCloseAllTabs": true,

```

Comment:
I'd probably remove this from this file - this file is just the "blank" `profiles.json` file we create when the Terminal is first run, and is intended to remain as minimal as possible (hence why very few of the other settings are in it).

Reply:
Done.